website/www/site/content/en/get-started/downloads.md

```diff
@@ -96,6 +96,14 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.64.0 (2025-03-25)
+
+Official [source code download](https://downloads.apache.org/beam/2.64.0/apache-beam-2.64.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.64.0/apache-beam-2.64.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.64.0/apache-beam-2.64.0-source-release.zip.asc).
+
+[Release notes](https://github.com/apache/beam/releases/tag/v2.64.0)
+
 ### 2.63.0 (2025-02-11)
 
 Official [source code download](https://downloads.apache.org/beam/2.63.0/apache-beam-2.63.0-source-release.zip).
```

Comment:
You need to change these links to archive links as well (should be same format as the 2.62.0 links below)

Reply:
You can see those links here - https://archive.apache.org/dist/beam/2.63.0/ - but https://downloads.apache.org/beam only contains the latest released version

Reply:
Fixed